.github/workflows/ci.yaml

```diff
@@ -0,0 +1,108 @@
+name: CI
+
+on:
+  push:
+    branches:
+    - use-github-actions
+
+env:
+  registry: ghcr.io
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: angular
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Check out the repository
+      id: checkout
+      uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+    - name: Set up node
+      id: setup-node
+      uses: actions/setup-node@1e60f620b9541d16bece96c5465dc8ee9832be0b # v4.0.3
+      with:
+        node-version: current
+        cache: npm
+        cache-dependency-path: angular/package-lock.json
+
+    - name: Run clean install
+      id: npm-ci
+      run: npm ci
+
+    - name: Run tests
+      id: npm-test
+      run: npm test -- --browsers ChromeHeadless --no-watch --no-progress
+
+    - name: Build the project
+      id: npm-build
+      run: npm run build -- --configuration production
+
+    - name: Set up buildx plugin
+      id: setup-buildx
+      uses: docker/setup-buildx-action@988b5a0280414f521da01fcc63a27aeeb4b104db # v3.6.1
+
+    - name: Log into registry ${{ env.registry }} as ${{ github.actor }}
+      id: docker-login
+      uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+      with:
+        registry: ${{ env.registry }}
+        username: ${{ github.actor }}
+        password: ${{ github.token }}
+
+    - name: Extract metadata
+      id: docker-meta
+      uses: docker/metadata-action@8e5442c4ef9f78752691e2d8f8d19755c6f78e81 # v5.5.1
+      with:
+        images: ${{ env.registry }}/${{ github.repository }}
+        tags: |
+          type=raw,value=latest
```

Comment:
Let's add possibility to pass tag as input and if it is not passed - use `latest`

Reply:
Done